container_pipeline/start_build.py

```diff
@@ -0,0 +1,75 @@
+import json
+import logging
+import os
+import sys
+
+import container_pipeline.utils as utils
+from container_pipeline.lib import settings
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+from container_pipeline.lib.queue import JobQueue
+
+
+def create_project(appid, jobid, repo_url, repo_branch, repo_build_path, target_file, notify_email, desired_tag, depends_on, test_tag):
```

Comment:
This function should either return ``True``/``False`` rather than ``0`` and ``1``.